maven-scm-providers/maven-scm-provider-local/src/main/java/org/apache/maven/scm/provider/local/command/update/LocalUpdateCommand.java

```diff
@@ -190,7 +190,7 @@ private List<ScmFile> update( File source, File baseDestination, List<File> file
 
             int chop = baseDestination.getAbsolutePath().length();
 
-            String fileName = "/" + destinationFile.getAbsolutePath().substring( chop + 1 );
+            String fileName = destinationFile.getAbsolutePath().substring( chop + 1 );
```

Comment:
Interesting that blindly a slash was prepended. That was doomed to fail on Windows, no?

Reply:
I have Ubuntu and it fails because the assertion expects it without a slash.

Reply:
And yes, this indeed a test that would probably fail on Windows too.